MvvmCross/Windows/Uwp/Views/MvxWindowsMultiRegionViewPresenter.cs

```diff
@@ -70,14 +70,14 @@ public override void Close(IMvxViewModel viewModel)
             }
         }
 
-        private static Type GetViewType(MvxViewModelRequest request)
+        protected static Type GetViewType(MvxViewModelRequest request)
         {
             var viewFinder = Mvx.Resolve<IMvxViewsContainer>();
             return viewFinder.GetViewType(request.ViewModelType);
         }
 
         // Implementation from: http://stackoverflow.com/a/1759923/80186
-        internal static T FindChild<T>(DependencyObject reference, string childName) where T : DependencyObject
+        protected internal static T FindChild<T>(DependencyObject reference, string childName) where T : DependencyObject
```

Comment:
@martijn00 I have not enough knowlege where this method is used MvvmCross near this class. So to prevent breaking applications I added protected and MvvmCross should work without any problems